index.html

```diff
@@ -46,14 +64,24 @@ <h3 class="subtitle">I’am web a developer.</h3>
             <img class="icon" src="images2/medium.png" alt="social-icon" />
           </li>
         </ul>
+
+      </div>
+    </div>
       </section>
       <section class="my-works1">
         <h1 class="blacktitle">My Recent works</h1>
+        <div class="line1">
+          <hr>
+          </div>
```

Comment:
- [ ] Please the title "My Recent Works" is aligned to the left and after there's a line that **takes the rest of the page's width**. Kindly adjust your design.
- Your design:
![la](https://user-images.githubusercontent.com/63972670/166094107-2d4e1dd1-714c-4a6c-933c-fee1f5de4a37.JPG)
- Original Design:
![lb](https://user-images.githubusercontent.com/63972670/166094124-f54dc200-40b2-45e4-bdac-d8a0fe887183.JPG)